lib/messenger.rb

```diff
@@ -0,0 +1,18 @@
+require 'dotenv'
+require 'twilio-ruby'
+
+class Messenger
+
+  def initialize
+    @account_sid = ENV['ACCOUNT_SID'] # Account SID from www.twilio.com/console
+    @auth_token = ENV['AUTH_TOKEN'] # Auth Token from www.twilio.com/console
+    @from = ENV['FROM'] # Your Twilio number
+    @to = ENV['TO'] # Your phone number
```

Comment:
This ENV naming is a bit suspect - you want to know what your env variables are for. In this specific case, I would ENV['TWILIO_PHONE_NUMBER'] and then pass an argument to the send_message method with the clients phone number. The clients phone number is sure to change, so storing as a constant seems wrong.